lib/loglevel.js

```diff
@@ -154,6 +164,6 @@
         return self;
     };
 
-    loadPersistedLevel();
+    self.setLevel(loadPersistedLevel() || "WARN", false);
```

Comment:
Can we rename this to getPersistedLevel, not load? Since the behaviour's now changed, and it's now just found and returned.